target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -78,6 +78,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	gb-pc1|\
```

Comment:
Albeit it will work this way, it is wrong. The gb-pc1 has only two ports attached to the switch according to your commit message. The case block is for a five port config.